mailet/test/src/main/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -413,7 +417,11 @@ public Object getAttribute(String name) {
 
     @Override
     public Iterator<String> getAttributeNames() {
```

Comment:
this method is not deprecated?

Reply:
Not because it's a `MailetContext`, and we do not have impacted the new Mailet API changes, maybe we should (in another migration campaign)

Reply:
ok, let's keep it for later